playbooks/utils/vm_reboot.yml

```diff
@@ -1,47 +1,35 @@
 ---
```

Comment:
On some playbooks we give examples on how to run it - would it be worthwhile to do that here, or is it a moot point since we'll be running it on Tower?

Reply:
I think it's a moot point, ignore this.